netbox/utilities/forms/fields/dynamic.py

```diff
@@ -66,6 +66,7 @@ class DynamicModelChoiceMixin:
             choice (DEPRECATED: pass `context={'disabled': '$fieldname'}` instead)
         context: A mapping of <option> template variables to their API data keys (optional; see below)
         selector: Include an advanced object selection widget to assist the user in identifying the desired object
+        quick_add: Include a widget to quickly create a new related object for assignment
```

Comment:
Maybe add a comment do not set it if the folllow-on form has any quick add fields.